### PR TITLE
ELTE Homepage update

### LINKED ISSUE #51: Auto skip ELTE "news" section.
According to this video: https://www.youtube.com/watch?v=C_bvX1CvgNM&list=PLm3AdVwMMCqwkassnBLieCx7LGM0buK4Z&index=1
After today update elte's neptun site going to have a 
* different login page
* a news site as default page after login, which requires an extra step to reach the actual neptun

Would it be acceptable adding elte specific changes to the NPU project ?
(I have access to ELTE Neptun, and can do test if needed)

### PR DESCRIPTION
**Pull request by leggenda47**
Supposed to resolve #51

### CODE REVIEW

**Review by Kokan — APPROVED** [review]
Thanks, this is great.

**Comment by gy-mate** [comment]
It works perfectly, thanks!
@solymosi could you review and merge it?

**Comment by solymosi** [comment]
Thanks a lot for the PR! 🎉 

My only issue is that we are adding _lots_ of ELTE-specific stuff to some quite central places in the codebase, which is going to reduce maintainability. Let me see if I can do some refactoring first.

**Comment by solymosi** [comment]
I will integrate this once this refactor is done & merged: https://github.com/solymosi/npu/pull/56

**Comment by whisperity** [comment]
> My only issue is that we are adding _lots_ of ELTE-specific stuff to some quite central places in the codebase

Isn't ELTE the biggest and the best university, though?

(And if the political situation continues as-is, will be the only one remaining...)

**Comment by solymosi** [comment]
With the 2.0 refactoring done we can now put all this into its own module and selectively enable it just on the ELTE domains. I'll land this soon.

File: npu.user.js [hunk 1]
```diff
@@ -1288,7 +1484,61 @@
...
+        const logoutButton = $("button:contains('Kilépés'), button:contains('Log out')");
+        logoutButton.click();
+      },
+
+      /*
+        Navigate to /Account/Login if the user is not logged in or
+        hallgato.neptun.elte.hu if the user is logged in.
+      */
+      navigateToELTELogin: function() {
+        window.location.href = "https://neptun.elte.hu/ToNeptunWeb/ToNeptunHWeb";
```
Comment (whisperity) [inline npu.user.js:1532]: @solymosi Please make sure that this thing here isn't hardcoded, though, or there is a way to specify which Neptun to auto log in to. Nowadays I use teacher web more often than student web and I'd not like if I'd have to manually go back either way... or if there is a way to grab if the user first opened `hallgato.neptun.elte.hu` or `oktato.neptun.elte.hu`, got redirected from there (can we catch and persist this piece of information?), and based on this, make a decision where to auto-redirect to from the script.

Comment (solymosi) [reply npu.user.js:1532]: No worries, I'll make the code more robust before merging and this is one of things that I'll clean up.

Comment (whisperity) [reply npu.user.js:1532]: The teacher Neptun old homepage is `oktato.neptun.elte.hu` and the relocating href is `/ToNeptunWeb/ToNeptun`**`O`**`Web`.

**Comment by solymosi** [comment]
Sorry, I simply don't have time to work on this. If anyone else wants to pick it up, feel free to submit another PR. For now I'll close it.

**Pull request closed without merging**